languages/python/exercises/concept/dicts/.docs/introduction.md

```diff
@@ -0,0 +1,6 @@
+A `dict` object in Python is a mutable\* mapping of hashable keys to values. A key is basically a name for a value. A value can be any type of object, like `strings`, `ints`, `lists`, etc. Dictionaries have different methods to change them, like getting a value from a specific key in the dictionary, setting a default value for a key and many more. Dictionaries are also iterable by their keys. For a full explanation of dictionaries in python refer to the [official documentation][docs] and on how to use them look at [W3-Schools.com'][how-to] tutorial.
+
+\*_Mutable objects are objects that can be changed without changing their identity. Methods for immutable objects actually dont return any value (like with strings) they change the object itself._
```

Comment:
Is there a concept document we can link to instead of putting this definition in line? This seems like information other exercises might require as well.

Reply:
I'm not sure, but if there is, we can certainly create one.

Reply:
There is a link :  https://github.com/exercism/v3/blob/master/languages/python/reference/concepts/builtin_types/dict.md.  However, it is good to also include any information here that is specific to only the learning objectives and solving the exercise (_like how dicts are made, how you reference a key or value, etc._), with code examples.  See [tuples concept exercise](https://github.com/exercism/v3/blob/master/languages/python/exercises/concept/tuples/.docs/introduction.md)   & [Elixr Tuples Exercise](https://github.com/exercism/v3/blob/master/languages/elixir/exercises/concept/maps/.docs/introduction.md).  Suggest that the following be moved to the `after.md`,  since it's valuable info..but not _directly_ applicable to completing the exercise, if that makes sense:

> Dictionaries have different methods to change them, like getting a value from a specific key in the dictionary, setting a default value for a key and many more. Dictionaries are also iterable by their keys. For a full explanation of dictionaries in python refer to the [official documentation][docs] and on how to use them look at [W3-Schools.com'][how-to] tutorial.

> \*_Mutable objects are objects that can be changed without changing their identity. Methods for immutable objects actually dont return any value (like with strings) they change the object itself._

Reply:
I'm not sure if I fully understand what you're saying, but do you mean that I should move text that you "quoted" to `after.md`? And then make the introduction look a little more like the one from the tuples exercise?

Reply:
@https://github.com/J08K -- Apologies - **yes** -- I think you should move the quoted text to the `after.md` document.  

The information is valuable for a student to know, but is too much for the `introduction.md`, which should focus only on information and examples the student would need to complete the tasks in the exercise.

Reply:
Ah yes, I understand. I am working on it now.

Reply:
What do you think of the new `after.md` and `introduction.md` now?